### DISCUSSION TITLE
Community extensions list in docs

### DISCUSSION THREAD

**Discussion by niklaswolf** [shopware/frontends#42] [Ideas]
I think it would be great to have a list of community extensions/modules/plugins related to Shopware Frontends in the docs.
There are already two Nuxt modules I know of, which add meaningful features:
- https://github.com/meeshoogendoorn/shopware-frontends-proxy by @meeshoogendoorn
- https://github.com/mothership-gmbh/nuxt-shopware-caching

New entries to the list can then be made through a PR.
I imagine having a little template to have a standardized way of describing these.
What do you think?

**Comment by elkmod** [comment]
I like the idea, however we should categorise these modules - similar to our [Internal Structure](https://shopware-frontends-docs.vercel.app/framework/internal-structure.html) documentation. Having in mind that some modules like the ones above depend on a Nuxt.js "runtime", others depend on having Tailwind.css as a CSS framework etc.

Opinions/ideas?

**Reply by niklaswolf** [reply]
yep, I think some sort of categorisation would be good. But I'm not quite sure about using the same categories as the internal structure. For example [my module](https://github.com/mothership-gmbh/nuxt-shopware-caching) would fall into more than one category, because it's a nuxt module and provides a composable...
I think a more broad categorisation would be totally sufficiant for now, e.g. "Nuxt modules", which would contain for example the two mentioned modules. It's then also more a responsibility of the modules READMEs to explain the dependencies etc.
I see the list more like a "hey, here are cool things you probably can use that the community build".

If the list then grows a lot, one can restructure it at a later point :)

**Comment by niklaswolf** [comment]
@elkmod added a site in the docs 🎉  https://shopware-frontends-docs.vercel.app/resources/community-modules/
so if you also have some related modules/plugins, you can now add those through a PR :)

**Reply by elkmod** [reply]
Please go here to add your own modules - https://github.com/shopware/frontends/blob/main/apps/docs/src/resources/community-modules/index.md
